distributed-commandbus-springcloud/src/main/java/org/axonframework/springcloud/commandhandling/MessageRoutingInformation.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2010-2017. Axon Framework
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.springcloud.commandhandling;
+
+import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.serialization.SerializedObject;
+import org.axonframework.serialization.Serializer;
+import org.axonframework.serialization.SimpleSerializedObject;
+
+import java.util.Objects;
+import java.util.function.Predicate;
+
+/**
+ * Object containing the message routing information required by the
+ * {@link org.axonframework.commandhandling.distributed.CommandRouter} to decide to whom an incoming
+ * {@link CommandMessage} should be routed.
+ * Holds the {@code loadFactor} and {@code commandFilter}, which respectively denote the desired load and set of
+ * CommandMessages a node can(not) handle.
+ *
+ * @author Steven van Beelen
+ */
+public class MessageRoutingInformation {
```

Comment:
It helps annotating this class as `implements Serializable` if it is intended to serialize this class (even if it's a bad idea to use Java Serialization).

Reply:
Added `implements Serializable`